content/en/platform/corda/5.2/developing-applications/cordapp-template/reset.md

```diff
@@ -0,0 +1,56 @@
+---
+date: '2023-11-01'
+title: "Resetting the Corda Runtime Gradle Plugin"
+description: Learn stop and clean the Corda runtime Gradle plugin.
+menu:
+  corda52:
+    parent: corda52-develop-get-started
+    identifier: corda52-reset
+    weight: 8000
+
+---
+# Resetting the Corda Runtime Gradle Plugin
+
+The CorDapp Template creates temporary files to store data required to generate and upload {{< tooltip >}}CPI{{< /tooltip >}} files and manage the Corda cluster.
+If these files are modified, deleted, or otherwise get out of sync with the actual state of the Corda cluster, the CorDapp Template Gradle tasks may not function correctly.
```

Comment:
```suggestion
If these files are modified, deleted, or otherwise get out of sync with the actual state of the Corda cluster, the CorDapp template Gradle tasks may not function correctly.
```